spec/factories/book_suggestion.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :book_suggestion do
+    editorial { Faker::Book.publisher }
+    price { Faker::Commerce.price }
+    author { Faker::Book.author }
+    title { Faker::Book.title }
+    link { Faker::Internet.url }
+    publisher { Faker::Book.publisher }
+    year { Faker::Number.between(1, 2018) }
+    user { create(:user) }
```

Comment:
Instead of creating an associated record manually, you can delegate this operation to the factory. Check [this link](https://www.rubydoc.info/gems/factory_bot/file/GETTING_STARTED.md#Associations)!